### PR TITLE
Add API parameter preventing AL feature to capture data for single request

### PR DESCRIPTION
**Pull request by PawelPeczek-Roboflow**
# Description
New feature letting us define `disable_active_learning` flag for a single request.

## Type of change

Please delete options that are not relevant.

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [x] This change requires a documentation update

## How has this change been tested, please provide a testcase or example of how you tested the change?

YOUR_ANSWER

## Any specific deployment considerations

For example, documentation changes, usability, usage/costs, secrets, etc.

## Docs

-   [x] Docs updated? What were the changes:

### CODE REVIEW

File: inference/core/managers/active_learning.py [hunk 1]
```diff
@@ -34,7 +35,14 @@ async def infer_from_request(
             model_id=model_id, request=request, **kwargs
         )
         active_learning_eligible = kwargs.get(ACTIVE_LEARNING_ELIGIBLE_PARAM, False)
-        if not active_learning_eligible or request.api_key is None:
+        active_learning_disabled_for_request = kwargs.get(
```
Comment (paulguerrie) [inline inference/core/managers/active_learning.py:38]: Should this be default False to match the default value on the request entities?

File: inference/core/managers/active_learning.py [hunk 2]
```diff
@@ -109,11 +117,18 @@ async def infer_from_request(
         self, model_id: str, request: InferenceRequest, **kwargs
     ) -> InferenceResponse:
         active_learning_eligible = kwargs.get(ACTIVE_LEARNING_ELIGIBLE_PARAM, False)
+        active_learning_disabled_for_request = kwargs.get(
```
Comment (paulguerrie) [inline inference/core/managers/active_learning.py:120]: Should this be default False to match the default value on the request entities?

**Review by paulguerrie — APPROVED** [review]
Looks good to me assuming I'm wrong in my comments and no action is actually required :-)

File: inference/core/managers/active_learning.py (hunk 1 above)
Comment (PawelPeczek-Roboflow) [reply inference/core/managers/active_learning.py:38]: it can be both as parameter haas default value, but right - I will change into `false`

**Review by paulguerrie — APPROVED** [review]

**Pull request merged**